game-core/src/main/java/games/strategy/engine/framework/startup/mc/GameSelectorModel.java

```diff
@@ -130,9 +122,13 @@ public synchronized boolean isSavedGame() {
   }
 
   private static void error(final String message, final Component ui) {
```

Comment:
Looks good.  The only suggestion I'd make is to annotate the `ui` parameter as `@Nullable`, since that's now a possibility.

Reply:
```@Nullable``` feels redundant at the moment. Until we resolve: https://github.com/triplea-game/triplea/issues/2885, everything is nullable by default : |

The benefit I see with the parameter annotation is for help within a method. It does not help a caller one way or another as everything is nullable... The explicit labelling I think is helpful when we want the IDE support for a longer method to help with nullity handling, or second when we have designed a class to be non-null by default and we are flagging the exception.

I'm not sure then if `@Nullable` is useful unless we make it consistent throughout the entire class.